js/src/templates/todolist.hbs

```diff
@@ -0,0 +1,5 @@
+<input class="listName" type="text" value="{{listName}}" /><button class="deleteList">Delete List</button>
+<ul></ul>
+<button class="addTodo">New Todo</button>
+
+
```

Comment:
Same comment about deleting trailing new lines and adding `js-` prefix if you'd like.